packages/flame/lib/src/game/camera/camera_wrapper.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:ui';
+import '../../../components.dart';
+import 'camera.dart';
+
+/// Do not use.
+///
+/// This class encapsulates BaseGame's rendering functionality. It will be
+/// converted into a proper Component in a future release, but until then
+/// using it in any code other than the BaseGame class is unsafe and
+/// not recommended.
+class CameraWrapper {
```

Comment:
how about CameraRenderer?

Reply:
Haha we have changed the name several times already, and it doesn't really matter, both since it is internal and since the new structure won't require this. :)

Reply:
sgtm!